proxy/core/src/main/scala/io/cloudstate/proxy/eventsourced/EventSourcedEntity.scala

```diff
@@ -253,9 +244,8 @@ final class EventSourcedEntity(configuration: EventSourcedEntity.Configuration,
     currentCommand =
       EventSourcedEntity.OutstandingCommand(idCounter, actorId + ":" + entityId + ":" + idCounter, sender)
     commandStartTime = System.nanoTime()
-    concurrencyEnforcer ! Action(currentCommand.actionId, () => {
-      relay ! EventSourcedStreamIn(EventSourcedStreamIn.Message.Command(command))
-    })
+    relay ! EventSourcedStreamIn(EventSourcedStreamIn.Message.Command(command))
+    statsCollector ! StatsCollector.CommandSent
```

Comment:
I think you can remove stats collector too... it's only used for autoscaling.

Reply:
@jroper Don't we want autoscaling? :)